docs/assets/joining_parallel_gateway_doc.bpmn

```diff
@@ -0,0 +1,233 @@
+    </bpmn:exclusiveGateway>
+    <bpmn:sequenceFlow id="Flow_00vq8qm" sourceRef="Activity_0kpi37x" targetRef="Gateway_01jetkt" />
+    <bpmn:task id="Activity_0zca2sa" name="ProcessInstance Element Activating">
+      <bpmn:incoming>Flow_0o85air</bpmn:incoming>
+      <bpmn:outgoing>Flow_1luh5vk</bpmn:outgoing>
+    </bpmn:task>
+    <bpmn:sequenceFlow id="Flow_0o85air" name="yes" sourceRef="Gateway_01jetkt" targetRef="Activity_0zca2sa" />
+    <bpmn:task id="Activity_182ap6o" name="Activate command rejected">
+      <bpmn:incoming>Flow_19jy9i0</bpmn:incoming>
+      <bpmn:outgoing>Flow_1hzq222</bpmn:outgoing>
+    </bpmn:task>
+    <bpmn:sequenceFlow id="Flow_19jy9i0" name="no" sourceRef="Gateway_01jetkt" targetRef="Activity_182ap6o" />
+    <bpmn:endEvent id="Event_0mm8r9p">
+      <bpmn:incoming>Flow_1hzq222</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:sequenceFlow id="Flow_1hzq222" sourceRef="Activity_182ap6o" targetRef="Event_0mm8r9p" />
+    <bpmn:task id="Activity_1360769" name="ProcessInstance Element Activated">
+      <bpmn:incoming>Flow_1luh5vk</bpmn:incoming>
+      <bpmn:outgoing>Flow_1qay9is</bpmn:outgoing>
+    </bpmn:task>
+    <bpmn:sequenceFlow id="Flow_1luh5vk" sourceRef="Activity_0zca2sa" targetRef="Activity_1360769" />
+    <bpmn:endEvent id="Event_1u4nt5u">
+      <bpmn:incoming>Flow_1qay9is</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:sequenceFlow id="Flow_1qay9is" sourceRef="Activity_1360769" targetRef="Event_1u4nt5u" />
+    <bpmn:textAnnotation id="TextAnnotation_1m4mds7">
+      <bpmn:text>Note: this command does more than just what is modelled here. This model only focuses on the parallel gateway</bpmn:text>
+    </bpmn:textAnnotation>
+    <bpmn:association id="Association_1sjz7k0" sourceRef="Activity_1nqfakp" targetRef="TextAnnotation_1m4mds7" />
+    <bpmn:textAnnotation id="TextAnnotation_11ton8q">
+      <bpmn:text>The number of taken sequence flows is stored in the element instance state. This contains the numberOfTakenSequenceFlowsColumnFamily.
+
+It gets stored using a key consisting of 3 parts:
+1. The flow scope key
+2. The gateway element id
+3. The sequence flow element id
+
+The value is an integer. This will result in a counter of how many times a specific incoming sequence flow of the gateway has been taken.</bpmn:text>
+    </bpmn:textAnnotation>
+    <bpmn:association id="Association_05nsw42" sourceRef="Activity_0np5087" targetRef="TextAnnotation_11ton8q" />
+    <bpmn:textAnnotation id="TextAnnotation_1s36nxm">
+      <bpmn:text>This is a method living in the state transition behavior. This is generally called when any element is completed.</bpmn:text>
+    </bpmn:textAnnotation>
+    <bpmn:association id="Association_14mumve" sourceRef="Event_1igkqew" targetRef="TextAnnotation_1s36nxm" />
+    <bpmn:textAnnotation id="TextAnnotation_17kdafp">
+      <bpmn:text>An ACTIVATE command is send. This is done regardless of the amount of sequence flows that have been taken</bpmn:text>
```

Comment:
I think you meant to say that the command is written. Only a user command is sent (e.g. using gRPC to the gateway, or by the gateway to the broker's command API).

🔧 Please consider using `is written` over `is send`.